### PR TITLE
add console presubmit job

### PR DESCRIPTION
**Pull request by yuswift**
1. `yarn && yarn test` to run the tests
2. `yarn && yarn build` to build 
Signed-off-by: yuswift <yuswift2018@gmail.com>

### CODE REVIEW

File: config/jobs/kubesphere/console/console-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+presubmits:
+  kubesphere/kubesphere:
+  - name: pull-console-unit-test
+    labels:
+      preset-go-build-cache: "true"
```
Comment (zryfish) [inline config/jobs/kubesphere/console/console-presubmits.yaml:5]: I think we can skip this for console.

File: config/jobs/kubesphere/console/console-presubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+presubmits:
+  kubesphere/kubesphere:
```
Comment (zryfish) [inline config/jobs/kubesphere/console/console-presubmits.yaml:2]: wrong repo

**Comment by yuswift** [comment]
yarn install v1.22.4
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@1.2.13: The platform "linux" is incompatible with this module.
info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > react-sortablejs@6.0.0" has unmet peer dependency "@types/sortablejs@^1.10.0".
warning " > babel-jest@23.6.0" has unmet peer dependency "babel-core@^6.0.0 || ^7.0.0-0".
[5/5] Building fresh packages...

**Comment by yuswift** [comment]
yarn install v1.22.4
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@1.2.13: The platform "linux" is incompatible with this module.
info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > react-sortablejs@6.0.0" has unmet peer dependency "@types/sortablejs@^1.10.0".
warning " > babel-jest@23.6.0" has unmet peer dependency "babel-core@^6.0.0 || ^7.0.0-0".
[5/5] Building fresh packages...
$ rimraf node_modules/.cache
Done in 142.23s.
yarn run v1.22.4
$ jest
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db

Why you should do it regularly:
https://github.com/browserslist/browserslist#browsers-data-updating
PASS src/components/Base/Dragger/index.test.js
PASS src/utils/index.test.js
PASS src/components/Inputs/EnvironmentInput/index.test.js
PASS src/utils/node.test.js
PASS src/components/Inputs/ContainerServicePort/index.test.js
PASS src/components/Inputs/ContainerPort/index.test.js
PASS src/components/Inputs/ArrayInput/index.test.js
PASS src/components/Base/ScrollLoad/index.test.js
PASS src/components/Inputs/S2iEnviroment/index.test.js
PASS src/components/Inputs/NumberInput/index.test.js
PASS src/components/Cards/Banner/index.test.js
PASS src/utils/base64.test.js
PASS src/components/Base/TypeSelect/index.test.js
PASS src/components/Base/List/index.test.js
PASS src/components/Base/Breadcrumb/index.test.js
PASS src/components/Inputs/ObjectInput/index.test.js
PASS src/components/Base/RadioGroup/index.test.js
PASS src/utils/devops.test.js
PASS src/components/Base/Bar/index.test.js
PASS src/components/Base/Avatar/index.test.js
PASS src/components/Base/Card/index.test.js
PASS src/components/Base/Modal/index.test.js
PASS src/components/Inputs/CardSelect/index.test.js
PASS src/components/Inputs/ReplicasInput/index.test.js
PASS src/components/Inputs/BoxSelect/index.test.js
PASS src/components/Base/Switch/index.test.js
PASS src/components/Base/Steps/index.test.js
PASS src/components/Base/Banner/index.test.js
PASS src/components/Inputs/BoolSelect/index.test.js
PASS src/utils/localStorage.test.js
PASS src/utils/form.persist.test.js
PASS src/components/Base/Empty/index.test.js
PASS src/utils/cookie.test.js

Test Suites: 33 passed, 33 total
Tests:       92 passed, 92 total
Snapshots:   0 total
Time:        24.485s
Ran all test suites.
Done in 27.04s.

**Comment by yuswift** [comment]
I've tested the prowjobs locally, I believe we can merge it now.

**Pull request merged**